plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java

```diff
@@ -74,6 +74,22 @@ public boolean isEnabled(Widget widget) {
 		}
 		return ret;
 	}
+	
+	/**
+	 * Finds out whether specified {@link Widget} is disposed or not.
+	 * 
+	 * @param widget widget to handle
+	 * @return true if widget is disposed, false otherwise
+	 */
+	public boolean isDisposed(final Widget widget) {
+		return Display.syncExec(new ResultRunnable<Boolean>() {
+
+			@Override
+			public Boolean run() {
```

Comment:
I disagree. Why would you pass a null widget ? I would consider it a mistake/error. I think we should throw exception. We do this for in other methods too so lets keep it consistent